verification/curator-service/ui/src/components/BulkCaseForm.test.tsx

```diff
@@ -30,7 +30,14 @@ afterEach(() => {
 });
 
 it('renders csv upload widget', async () => {
-    const { getByTestId, getByText } = render(<BulkCaseForm user={user} />);
+    const { getByTestId, getByText } = render(
+        <BulkCaseForm
+            user={user}
+            onModalClose={(): void => {
+                console.log('Closed!');
```

Comment:
You can leave this as an empty method `(): void => {}` to avoid console clutter

Reply:
There's a default Typescript eslint rule that [yells at you for empty functions](https://github.com/typescript-eslint/typescript-eslint/blob/v2.34.0/packages/eslint-plugin/docs/rules/no-empty-function.md). But I can swap this out for a non-logging noop variant.